Mlem/Views/Shared/Markdown View.swift

```diff
@@ -201,8 +202,42 @@ private extension Color {
 struct MarkdownView: View {
 
     @State var text: String
+    let imageProvider: CachedImageProvider
 
     var body: some View {
+        generateView()
+    }
+
+    @MainActor func generateView() -> some View {
+
+        let imageLooker = Regex {
+            "!["
+            Capture {
+                ZeroOrMore(.any, .reluctant)
+            }
+            "]("
+            Capture {
+                ZeroOrMore(.any, .reluctant)
+            }
+            ")"
+        }
```

Comment:
Nit, but for readability even simple regexes should have an explanatory comment

Reply:
Added comments!